exercises/forth/canonical-data.json

```diff
@@ -213,6 +213,24 @@
             "instructions": ["2 4 * 3 /"]
           },
           "expected": [2]
+        },
+        {
+          "uuid": "f749b540-53aa-458e-87ec-a70797eddbcb",
+          "description": "multiplication and addition",
+          "property": "evaluate",
+          "input": {
+            "instructions": ["1 3 4 * +"]
+          },
+          "expected": [13]
```

Comment:
Would it make sense to simplify this and only do one operation?
`["1 3 4 *] => [1, 12]`

Reply:
Yes. In that case, I should do one for each of `+`, `-`, `*` and `/`. These tests are in the "combined arithmetic". I'll rework this.

Reply:
Do the combined arithmetic tests add anything that isn't already captured by prior tests? If we test all the operators when there are both 2 and 3 values on the stack, the combined tests shouldn't be adding any additional coverage, I think?

Reply:
I think they are enhancing the "combined arithmetic" tests, in that it's OK to load the stack with numbers first and then perform multiple operations to consume/transform the stack.